templates/partials/alert.html

```diff
@@ -0,0 +1,3 @@
+<div class="col-12 position-sticky mt-5 construction-alert">
+        <p class="m-auto text-center fs-3">Check back for more updates!</p>
```

Comment:
#30 points out that there should be some mention about the project status. This doesn't really convey that information

What about:  "The Marble project is in active development. Check back here for updates."

Reply:
That sums it up.

Changed the text to the message.